app/redux/src/ReduxContext.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { PersistGate } from 'redux-persist/es/integration/react';
+import { Provider } from 'react-redux';
+import { type Reducer } from 'redux';
+import { persistStore } from 'redux-persist';
+
+import Store, { createCombinedReducer } from './Store';
+
+type Props = {|
+  children: React.Node,
+  reducers: { [string]: Reducer },
+|};
+
+/**
+ * Creates Redux context for underlying components. It makes only sense if
+ * you pass additional reducers to be registered.
+ */
+export default class ReduxContext extends React.Component<Props> {
+  componentWillMount = () => {
+    // it's too late to register new reducers in componentDidMount
```

Comment:
I'm not sure if I comprehend this comment, why it's then done in this place? :)

Reply:
Because `componentWillMount` is called before `componentDidMount` (too late). I could add it also to the constructor...

You can maybe delete this comment if it's not necessary.

Reply:
dammit, I saw `componentDidMount` instead `componentWillMount`, that's why I was confused 🤕